lib/coins.py

```diff
@@ -70,6 +70,7 @@ class Coin(object):
     BLOCK_PROCESSOR = BlockProcessor
     XPUB_VERBYTES = bytes('????', 'utf-8')
     XPRV_VERBYTES = bytes('????', 'utf-8')
+    BASE58_CHECK_HASH = double_sha256
```

Comment:
I think I'd prefer ENCODE_CHECK=Base58.encode_check.

Then decred can use a partial.  What do you think?

However a way to avoid unconditionally importing the new hash lib needs to be found....